Widgets/LockScreenWidgets.swift

```diff
@@ -119,6 +118,23 @@ struct PasswordsLockScreenWidget: Widget {
     }
 }
 
+/// Waiting copywriting and icon for this.
```

Comment:
Decided to leave it here, you can already test it by adding the AIChatLockScreenWidget into the Widgets.swift